src/utils/drawHeader/drawRows/drawWeeksInMiddle.ts

```diff
@@ -25,7 +24,9 @@ export const drawWeeksInMiddle = (
   let xPos = 0;
 
   for (let i = 0; i < weeksThreshold; i++) {
-    const day = dayjs(`${startDate.year}-${startDate.month + 1}-${startDate.dayOfMonth}`).day();
+    const formattedDate = `${startDate.year}-${startDate.month + 1}-${startDate.dayOfMonth}`;
```

Comment:
Can't it be like this?
```suggestion
    const formattedDate = dayjs(startDate).format('YYYY-M-D');
```

Reply:
Thanks for comment! Unfortunately not, startDate is a custom `Day` type not compatible with dayjs